test/integration_test.rb

```diff
@@ -0,0 +1,62 @@
+require 'helpers/kubeclient_helper'
+
+module KubernetesDeploy
+  class IntegrationTest < ::Minitest::Test
+    include KubeclientHelper
+
+    def setup
+      @logger_stream = StringIO.new
+      @logger = Logger.new(@logger_stream)
+      KubernetesDeploy.logger = @logger
```

Comment:
Yep, that's because there are two loggers and the one that does kube status is not at all pluggable right now. We should definitely address it, but I'd rather leave it for later since it involves modifying the non-test code (also, while crafting tests, that output is useful anyhow :P).